models/events.sql

```diff
@@ -17,116 +17,139 @@ select
                     "name": "device_user_agent",
                     "path": "$.user_agent.raw_user_agent",
                     "cast_as": "string",
+                    "skip_parse": var("fullstory_skip_json_parse", False),
```

Comment:
High level question: why don't we use an intermediate CTE to get a JSON column that we can then just address like normal json? Then we wouldn't have to customize how every field is fetched.

Reply:
That's a good idea! This `parse_json_into_columns` has existed for a long time so just kind of kept going with it. We'd need to evaluate how that CTE performed in an incremental situation. I suppose we'd be able to move the incremental logic to the CTE.

Reply:
@camphillips22 I think that change is slightly out of scope of this PR, but we will track it as an improvement to be made internally.

Reply:
Sounds good. For now, this works, just isn't super fun to maintain.